docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
+services:
+  bot:
+    build: .
+    depends_on: ["redis"]
```

Comment:
Would be better to do
```
depends_on:
  - redis
```
This would be more inline with commonly used syntax and easier to update in the future as we add insane lists of dependencies!